docs/reference/commandline/service_create.md

```diff
@@ -47,16 +47,16 @@ Options:
       --name string                      Service name
       --network list                     Network attachments (default [])
       --no-healthcheck                   Disable any container-specified HEALTHCHECK
-  -p, --publish list                     Publish a port as a node port (default [])
-      --replicas uint                    Number of tasks (default none)
+  -p, --publish port                     Publish a port as a node port
```

Comment:
Should we change this one to show "list" as well (not in this PR) @dnephin? We may want to find a way to express which flags can be set multiple times